cmd/neofs-cli/internal/client/sdk.go

```diff
@@ -29,33 +28,37 @@ func GetSDKClientByFlag(cmd *cobra.Command, key *ecdsa.PrivateKey, endpointFlag
 		common.ExitOnErr(cmd, "", fmt.Errorf("%v: %w", errInvalidEndpoint, err))
 	}
 
-	cli, err := GetSDKClient(context.TODO(), cmd, key, addr)
+	cli, err := GetSDKClient(cmd, key, addr)
 	if err != nil {
 		common.ExitOnErr(cmd, "can't create API client: %w", err)
 	}
 	return cli
 }
 
 // GetSDKClient returns default neofs-sdk-go client.
-func GetSDKClient(ctx context.Context, cmd *cobra.Command, key *ecdsa.PrivateKey, addr network.Address) (*client.Client, error) {
+func GetSDKClient(cmd *cobra.Command, key *ecdsa.PrivateKey, addr network.Address) (*client.Client, error) {
```

Comment:
But why removing `ctx` from it? We can have it as a parameter, especially given that it's already there.

Reply:
this corrects the code that passed `context.TODO()` and simplifies usage cuz all callers pass `cmd.Context()` anyway

Reply:
https://pkg.go.dev/context
> Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx:

I know we violate this right and left, but at least we can try not making the situation worse (these is `ctx` already).

Reply:
i'd say cobra violates `Do not store Contexts inside a struct type` and we just follow this, but ok, i'll leave context as prm

Reply:
done